sdk/storage/azure-storage-blob-nio/src/main/java/com/azure/storage/blob/nio/AzureFileSystemProvider.java

```diff
@@ -373,7 +374,8 @@ public OutputStream newOutputStream(Path path, OpenOption... options) throws IOE
         // Write and truncate must be specified
         if (!optionsList.contains(StandardOpenOption.WRITE)
             || !optionsList.contains(StandardOpenOption.TRUNCATE_EXISTING)) {
-            throw new IllegalArgumentException("Write and TruncateExisting must be specified to open an OutputStream");
+            throw LoggingUtility.logError(logger,
```

Comment:
We should update other locations to using `LoggingUtility` when throwing exceptions